src/metalnx-web/src/main/resources/views/collections/fileInfo.html

```diff
@@ -40,7 +40,7 @@
 								title="Permissions">Permissions</span>
 						</a></li>
 
-						<li><a href="#preview" data-toggle="tab" role="tab"
+						<li th:if="${permissionType == 'read' or permissionType == 'write' or permissionType == 'own'}"><a href="#preview" data-toggle="tab" role="tab"
 							th:onclick="'javascript:getPerview(\'' + ${dataProfile.absolutePath} + '\');'">
```

Comment:
was this always 'Perview'?   might need to find/replace that wherever else it is too...

Reply:
I think this is the 'preview' tab when you click 'view info' button on a file. If you click on a collection, this template will not be used. Not sure if there is somewhere else that should use this logic, I can poke around.

Reply:
i just meant the typo itself... apparently used in one other place in the code.

"Perview" -> "Preview"

Reply:
done. replaced at two places.